src/main/java/milk/products/iot/Milk.java

```diff
@@ -0,0 +1,43 @@
+package milk.products.iot;
+
+import java.util.Objects;
+
+public final class Milk extends MilkProduct {
+    private double percentageOfFat;
+    private boolean lactoseFree;
+
+    public Milk(final String nameOfMilkProduct, final String origin, final double volumeOrMass, final double priceInUahPerKilo, final boolean homeMade, final double percentageOfFat, final boolean lactoseFree) {
+        super(nameOfMilkProduct, origin, volumeOrMass, priceInUahPerKilo, homeMade);
```

Comment:
Pay attention to line-wrapping.

Here's what Oracle documentation says:
>**4.1 Line Length**
>_Avoid lines longer than 120 characters, since they're not handled well by many terminals and tools._